.github/.github_changelog_generator

```diff
@@ -1,11 +1,11 @@
 issues=false
 pr-wo-labels=false
 author=false
-breaking-label=Changed
+breaking-label=**Changed:**
 breaking-labels=Breaking change
-enhancement-label=Added
+enhancement-label=**Added:**
 enhancement-labels=Added
-bugs-label=Patch
+bugs-label=**Patch:**
 bug-labels=Patch
 deprecated-labels=Deprecated
 removed-labels=Removed
```

Comment:
We should also specify `deprecated-label` and `removed-labels` to avoid differences with the default if we decide to change again (and for clarity).

E.g.
```
breaking-label=**Changed**
breaking-labels=Breaking change
enhancement-label=**Added**
enhancement-labels=Added
bugs-label=**Patch**
bug-labels=Patch
deprecated-labels=Deprecated
deprecated-label=**Deprecated**
removed-labels=Removed
removed-label=**Removed**
```

Reply:
> We should also specify `deprecated-label` and `removed-labels` to avoid differences with the default if we decide to change again (and for clarity).
> 
> E.g.
> 
> ```
> breaking-label=**Changed**
> breaking-labels=Breaking change
> enhancement-label=**Added**
> enhancement-labels=Added
> bugs-label=**Patch**
> bug-labels=Patch
> deprecated-labels=Deprecated
> deprecated-label=**Deprecated**
> removed-labels=Removed
> removed-label=**Removed**
> ```

done in 571d2101cd24749157dccbad6b1e0d4f506d98c4